docs/ops.html

```diff
@@ -1972,6 +1972,27 @@ <h4 class="anchor-heading"><a id="tiered_storage_monitoring" class="anchor-link"
         <td>Average idle percent of thread pool for copying data to remote storage</td>
         <td>kafka.log.remote:type=RemoteLogManager,name=RemoteLogManagerTasksAvgIdlePercent</td>
       </tr>
+      <tr>
+        <td>RemoteLogManager Avg Broker Throttled Fetch Time</td>
+        <td>The average time in millis remote fetches was throttled by a broker</td>
+        <td>kafka.log.remote:type=RemoteLogManager, name=remote-fetch-throttle-time-avg	</td>
```

Comment:
@Nancy-ksolves This should be `kafka.server:type=RemoteLogManager` as per the discussion here https://github.com/apache/kafka/pull/16087#issuecomment-2202296076

Same for the other metrics

Reply:
Hi @abhijeetk88,
We need to update kafka.log.remote:type=RemoteLogManager to kafka.server:type=RemoteLogManager in the documentation. However, in the [KAFKA-17313](https://issues.apache.org/jira/browse/KAFKA-17313), the metric object name for tiered storage quotas metrics is set to kafka.server:type=RemoteLogManager instead of kafka.log.remote:type=RemoteLogManager. Could you please confirm if we need to make this change in the PR?

Reply:
@Nancy-ksolves :
1. [KAFKA-17313](https://issues.apache.org/jira/browse/KAFKA-17313) is the ticket we created for future improvement. That will not be included in the coming v3.9.0 release.
2. What we want to do here, before [KAFKA-17313](https://issues.apache.org/jira/browse/KAFKA-17313) completed, is to let the document align with what we implemented, which is `kafka.server:type=RemoteLogManager`. 

Let us know if it's not clear. Thanks.